server/configuration.js

```diff
@@ -20,6 +22,11 @@ export const setEnvironment = () => {
             process.env.TRAY_ENDPOINT = productionGraphqlEndpoint;
             process.env.TRAY_APP_URL = appUrlProd;
             break;
+        case 'eu1-prod':
```

Comment:
question why s there two cases for eu prod?

Reply:
It's just to match the others, the other env's have two cases which look like to be a full name and a shortened name